tools/monitoring/config.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"log"
+	"os"
+	"strconv"
+)
+
+var databaseConfig MySQLConfig
+
+func init() {
+	var err error
+	databaseConfig, err = configureMonitoringDatabase()
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func configureMonitoringDatabase() (MySQLConfig, error) {
+	var config MySQLConfig
+
+	port, err := strconv.Atoi(os.Getenv("MONITORING_DB_PORT"))
+	if err != nil {
+		return config, err
+	}
+
+	config = MySQLConfig{
+		DatabaseName: os.Getenv("MONITORING_DB_NAME"),
+		Username:     os.Getenv("MONITORING_DB_USER"),
+		Password:     os.Getenv("MONITORING_DB_PASSWORD"),
```

Comment:
this should be the secret, correct? Then we should mount this and not part of env var

Reply:
Done